yt_dlp/extractor/dropout.py

```diff
@@ -138,7 +138,7 @@ def _real_extract(self, url):
         return {
             '_type': 'url_transparent',
             'ie_key': VHXEmbedIE.ie_key(),
-            'url': embed_url,
+            'url': VimeoIE._smuggle_referrer(embed_url, 'https://www.dropout.tv'),
```

Comment:
Since we are passing it to VHXEmbed, move `_smuggle_referrer` to `VimeoBaseInfoExtractor` and use `VHXEmbedIE._smuggle_referrer`